packages/react-sdk/src/components/common/MenuItemSwitch/MenuItemSwitch.test.tsx

```diff
@@ -18,13 +18,19 @@ import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
 import { Menu } from '@mui/material';
 import { act, render, screen, within } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { axe } from 'jest-axe';
+import axe from 'axe-core';
 import { ComponentType, PropsWithChildren } from 'react';
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest';
 import { MenuItemSwitch } from './MenuItemSwitch';
 
 describe('<MenuItemSwitch>', () => {
-  const onChange = jest.fn();
-  const onClick = jest.fn();
+  const onChange = vi.fn();
+  const onClick = vi.fn();
+
+  afterEach(() => {
+    onChange.mockReset();
```

Comment:
See other comment. If we need a fresh mock on each run, maybe put it into a `beforeEach` instead